targets/python/templates/API.py.ejs

```diff
@@ -0,0 +1,34 @@
+import PlayFabErrors
+import PlayFabHTTP
+import json
+import PlayFabSettings
+
+<%- generateApiSummary("", api, "description")%>
+<% for(var i in api.calls) { var apiCall = api.calls[i];
+%><%- generateApiSummary("", apiCall, "summary")
+%><%- getDeprecationAttribute("", apiCall)
+%>def <%- apiCall.name %>(request, customData, extraHeaders, postProcessingFunc):
+<%- getRequestActions("    ", apiCall) %>
+    playfabResult = PlayFabHTTP.DoPost("<%- apiCall.url %>", request, <%- getAuthParams(apiCall) %>, extraHeaders)
+    if type(playfabResult) is PlayFabErrors.PlayFabError:
+        error = playfabResult
+        postProcessingFunc(None, error)
+        return error
+
+    returnResult = playfabResult
```

Comment:
I think that once we fix the getResultActions function in make.js, then we have no need to reassign this variable name.
We can just keep it named playfabResult for the rest of the function, and remove/convert returnResult